src/components/ViewMenu.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { Table } from "semantic-ui-react";
+import { getMenu } from "../modules/menu";
+import { Component } from "react";
+
+class ViewMenu extends Component {
+  state = {
+    menu: [],
+  };
+  componentDidMount() {
+    this.getViewMenu();
+  }
+  async getViewMenu() {
+    let result = await getMenu();
+    this.setState({ menu: result.data.products });
+  }
+  render() {
+    let menu;
+    if (this.state.menu != null) {
```

Comment:
Is an empty array null? You could do use the `.empty?` also, I think. But if the test is passing I guess `!= null` works fine!

Reply:
Yeah put a debugger here and try this out. I do not think that an empty array is equal to null.

You could try `!= []` instead

Reply:
Removed the sad path, and no check if the response from backend is empty